dev/src/main/resources/messages/ui/CodewindUIBundle.properties

```diff
@@ -238,6 +240,10 @@ RefreshProjectJobLabel=Refreshing project: {0}
 	
 ImportProjectError=An error occurred while importing the {0} project.
 StartBuildError=An error occurred while starting a build for the {0} project: {1}.
+SettingUpProject=Setting up project and dependencies
+PleaseWaitForBuild=Please wait while dependencies are downloaded
+NewProjectSetupProcessCancelledMessage=The project set up process was cancelled. The project was not created.
+NewProjectSetupProcessCancelledTitle=Project Set Up Cancelled
```

Comment:
- "setup" should be one word.
- "canceled" with one "l" is the preferred American spelling, so please change to that.

Reply:
Done